components/count-badge/count-badge-icon.js

```diff
@@ -0,0 +1,108 @@
+import '../colors/colors.js';
+import '../icons/icon.js';
+import { CountBadgeMixin, countBadgeStyles } from './count-badge-mixin.js';
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { ifDefined } from 'lit-html/directives/if-defined.js';
+import { offscreenStyles } from '../offscreen/offscreen.js';
+import { RtlMixin } from '../../mixins/rtl-mixin.js';
+import { styleMap } from 'lit-html/directives/style-map';
+
+class CountBadgeIcon extends CountBadgeMixin(RtlMixin(LitElement)) {
+
+	static get properties() {
+		return {
+			/**
+			 * The icon string to appear with the badge.
+			 * @type {string}
+			 */
+			icon: {
+				type: String,
+				reflect: true
+			},
+			_forceTooltipOn: {
+				type: Boolean
+			},
+		};
+	}
+
+	static get styles() {
+		return [offscreenStyles, countBadgeStyles, css`
+		:host(.focus-visible) d2l-icon,
+		.d2l-count-badge-wrapper.focus-visible d2l-icon {
+			color: var(--d2l-color-celestine);
+		}
+
+		.d2l-count-badge-number {
+			position: relative;
+		}
+
+		.d2l-count-badge-wrapper {
```

Comment:
This and the other rules targeting elements/classes defined by the mixin feels like a little bit of a leaky abstraction... where the consumer of the mixin (count badge icon) knows things about the makup of the mixin itself.

There are a couple ways I can think of to get around this:
1. Expose these as CSS variables on the mixin (e.g. `--d2l-count-badge-wrapper-padding`) and then the consumer here would set that `:host { --d2l-count-badge-wrapper-padding: 0 14px 0 2px; }`.
2. Pass these things in as parameters to the `renderCount()` method.